case-study-orchestral/index.html

```diff
@@ -0,0 +1,2648 @@
+<!DOCTYPE html>
+<html lang="en-US" prefix="og: https://ogp.me/ns#">
+  <head>
+    <style type="text/css">
+      .ttfm1 {
+        font-family: "dual300";
+        font-size: em;
+        line-height: em;
+        color: black;
+      }
+      .ttfm2 {
+        font-family: "Roboto";
+        font-size: em;
+        line-height: 3em;
+        color: black;
+      }
+    </style>
+
+    <!--[if lt IE 9]>
+      <script src="/wp/wp-content/themes/rise/js/html5/dist/html5shiv.js"></script>
+      <script src="//css3-mediaqueries-js.googlecode.com/svn/trunk/css3-mediaqueries.js"></script>
+    <![endif]-->
+    <!--[if IE 8]>
+      <link
+        rel="stylesheet"
+        type="text/css"
+        href="/wp/wp-content/themes/rise/css/ie8.css"
+      />
+    <![endif]-->
+    <!--[if IE 7]>
+      <link
+        rel="stylesheet"
+        type="text/css"
+        href="/wp/wp-content/themes/rise/css/ie7.css"
+      />
+    <![endif]-->
+
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- WP_HEAD() START -->
+    <script type="text/javascript">
+      function apbct_attach_event_handler__backend(elem, event, callback) {
+        if (typeof window.addEventListener === "function")
+          elem.addEventListener(event, callback);
+        else elem.attachEvent(event, callback);
+      }
+      apbct_attach_event_handler__backend(window, "load", function () {
+        if (typeof apbctLocalStorage === "object")
+          apbctLocalStorage.set("ct_checkjs", "232650532", false);
+        else console.log("APBCT ERROR: apbct-public--functions is not loaded.");
+      });
+    </script>
+    <!-- Search Engine Optimization by Rank Math - https://s.rankmath.com/home -->
+    <title>Case Study: Orchestral - StackStorm</title>
+    <meta
+      name="robots"
+      content="follow, index, max-snippet:-1, max-video-preview:-1, max-image-preview:large"
+    />
+    <link
+      rel="canonical"
+      href="https://orchestral.local/orchestral-stackstorm-page/"
+    />
+    <meta property="og:locale" content="en_US" />
+    <meta property="og:type" content="article" />
+    <meta
+      property="og:title"
+      content="Orchestral StackStorm Page - Orchestral.ai"
+    />
+    <meta
+      property="og:url"
+      content="https://orchestral.local/orchestral-stackstorm-page/"
+    />
+    <meta property="og:site_name" content="Orchestral.ai" />
+    <meta property="article:author" content="admin@orchestral.ai" />
+    <meta property="og:updated_time" content="2022-07-31T17:37:14-07:00" />
+    <meta
+      property="og:image"
+      content="https://orchestral.local/wp-content/uploads/2021/01/Orchestral.ai_full_logo.png"
```

Comment:
I guess what's left is to transfer the images to this PR and change all the URLs from the `orchestral.local` on this page.

Reply:
Thanks Eugen,
I'll just change the `orchestral.local` URLs to point to the images on the Orchestral website given that they are already hosted there.

Reply:
That would work technically, but practically stackstorm.com should not reference the externally hosted images.
So there's still a need to mirror those copies at stackstorm.com website.

Reply:
Yeah, you're correct about referencing externally hosted images. 
I thought about this after I sent my comment but definitely it would be "best practice" to have the images hosted within GHP.

So, I assume by your comment - 
"...transfer the images to this PR to include them in git and change all the URLs.."

That one way to accomplish this would be to create an "images" sub-directory under the "case-study-orchestral" directory like this:

root--|--case-study-orchestral--|--images

Then, I just dump all the images into this "images" sub-directory and change the URLs accordingly....and done!

Agreed?